extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeFooter.php

```diff
@@ -4,13 +4,17 @@
 class NodeFooter extends Node {
 
 	public function getData() {
-		return [ 'value' => $this->getExternalParser()->parseRecursive(
-			\Wikia\PortableInfobox\Helpers\SimpleXmlUtil::getInstance()->getInnerXML( $this->xmlNode )
-		) ];
+		if ( !isset( $this->data ) ) {
+			$this->data = [ 'value' => $this->getInnerValue( $this->xmlNode ) ];
+		}
+
+		return $this->data;
 	}
 
-	public function isEmpty( $data ) {
-		$links = trim( $data['value'] );
+	public function isEmpty() {
+		$data = $this->getData();
+		$links = trim( $data[ 'value' ] );
+
 		return empty( $links ) && $links != '0';
```

Comment:
I know it wasn't changed in this PR, but this line looks like duplicate logic as compared to 
https://github.com/Wikia/app/pull/7467/files#diff-b09ea7c9a303b75f3ed56e8a326e0e0cR100
if my comment there makes sense